<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn_choose_file">
      <property name="text">
       <string>Choose file</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn_load_data">
      <property name="text">
       <string>Load Data</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2">
     <widget class="QGroupBox" name="move">
      <property name="title">
       <string>Move</string>
      </property>
      <widget class="QPushButton" name="move_down">
       <property name="geometry">
        <rect>
         <x>92</x>
         <y>26</y>
         <width>40</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>down</string>
       </property>
      </widget>
      <widget class="QPushButton" name="move_left">
       <property name="geometry">
        <rect>
         <x>55</x>
         <y>24</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>left</string>
       </property>
      </widget>
      <widget class="QPushButton" name="move_right">
       <property name="geometry">
        <rect>
         <x>135</x>
         <y>25</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>right</string>
       </property>
      </widget>
      <widget class="QPushButton" name="move_up">
       <property name="geometry">
        <rect>
         <x>95</x>
         <y>-6</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>up</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="4" column="0" colspan="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="3" rowspan="2">
     <widget class="QGroupBox" name="scale">
      <property name="title">
       <string>Scale</string>
      </property>
      <widget class="QPushButton" name="scale_dec">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>-</string>
       </property>
      </widget>
      <widget class="QPushButton" name="scale_inc">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>10</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="text">
        <string>+</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="QGroupBox" name="rotate">
      <property name="title">
       <string>Rotate</string>
      </property>
      <widget class="QPushButton" name="rotate_up">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>0</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>up</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotate_left">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>left</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotate_right">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>right</string>
       </property>
      </widget>
      <widget class="QPushButton" name="rotate_down">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>35</width>
         <height>35</height>
        </rect>
       </property>
       <property name="text">
        <string>down</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
